tests/performance/lower_upper_function.xml

```diff
@@ -0,0 +1,11 @@
+<test>
+    <query>select lower(randomString(16))</query>
```

Comment:
I guess it should be something like `select lower(data) from generateRandom('data String') limit 1e6 format Null` or similar, since it is pretty hard to see the difference in one function call.